packages/gatsby-plugin-eufemia-theme-handler/themeHandler.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import { Theme } from '@dnb/eufemia/shared'
```

Comment:
Seem like this import causes the following issue:
```
 ERROR #98124  WEBPACK.DEVELOP-HTML

Generating development SSR bundle failed

Can't resolve '@dnb/eufemia/shared' in '/Users/anderslangseth/dev/eufemia/node_modules/gatsby-plugin-eufemia-theme-handler'

If you're trying to use a package make sure that '@dnb/eufemia/shared' is installed. If you're trying to use a local file make sure that the path is correct.

File: ../../node_modules/gatsby-plugin-eufemia-theme-handler/themeHandler.js:2:0
```

Also reported here: https://github.com/dnbexperience/eufemia/pull/2671/files#r1339666868